Cargo.toml

```diff
@@ -29,7 +29,7 @@ serde-1 = ["serde", "serde_derive"] # enables serialisation for PRNGs
 members = ["rand_core"]
 
 [dependencies]
-rand_core = { version = '0.1.0-pre.0', default-features = false }
+rand_core = { path="rand_core", default-features = false }
```

Comment:
An actual version is needed for making a release of `rand` itself. Are there changes in `rand_core` that mean we cannot use the pre-release now?

Reply:
I expected `rand_core` would be released first as 0.1, and shortly after that `rand` 0.5. Or do you want to make 0.5 depend on a pre-release version?

Reply:
Sorry, this comment should be in #366.

Do we have any changes to `rand_core` besides a few documentation tweaks? I don't think so, so I don't see why we can't still use the pre-release here and update that to a 0.1 release some time before 0.5?

Reply:
I am not sure I follow. Do you want to do another pre-release of Rand? Or do you want to keep building with the crates.io version of `rand_core` on master? In that case we would have to disable `cargo-deadlinks` on the CI.